libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/ConwayGovActionReorder.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Examples.ConwayGovActionReorder (
```

Comment:
This test seems like it could live in `cardano-ledger-conway`,  next to the new tests, so in 
`eras/conway/impl/test/Test/Cardano/Ledger/Conway`
In this way it could use `Test.Cardano.Ledger.Common` and HSpec instead of Tasty . 
Or if not, in cardano-ledger-conway-test, but from what I understand, the plan is to eventually move the tests from there into cardano-ledger-conway. 

But not sure, maybe you had a good reason to add it to `cardano-ledger-test` ?

Reply:
I thought we were moving our tests to `cardano-ledger-test`, but it could just as well live in `cardano-ledger-conway`

Reply:
> I thought we were moving our tests to cardano-ledger-test

@Soupstraw This is quite the opposite. Only tests that cannot live in a test suite for specific era, should go into `cardano-ledger-test`. In fact we alredy have too many tests in `cardano-ledger-test` that could be moved into a specific era, but we'll deal with that later.

However, going forward, we should try our best to implement tests in `cardano-ledger-[era]:tests`, instead of `cardano-ledger-[era]-test` or `cardano-ledger-test`.

The ultimate goal is to get rid of `cardano-ledger-[era]-test` packages completely and make `cardano-ledger-test` as thin as possible, which means only test that truly require the `Proof` or other things that are unique to that package, like constraint based testing, will live there.

See #3623 for a bit more context.

Reply:
`PrettyA` instances are often the reason I move tests to `cardano-ledger-test`, but I haven't used pretty printing here yet.

Reply:
We should also be switching to ToExpr instances from PrettyA as well for testing